<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="originalImage">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>551</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="segmentedImage">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>551</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Segmented Image</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>310</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="drawContourButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>10</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>draw contour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>143</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="meanshiftButton">
      <property name="text">
       <string>Meanshift</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="kmeansComboBox">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="kmeansButton">
      <property name="text">
       <string>Kmeans</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="gmmButton">
      <property name="text">
       <string>GMM</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="runKmeansButton">
      <property name="text">
       <string>Start Kmeans !</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="regionGrowingButton">
      <property name="text">
       <string>RegionGrowing</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="watershedButton">
      <property name="text">
       <string>Watershed(opencv)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="grabcutButton">
      <property name="text">
       <string>Grabcut(opencv)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="meanshiftComboBox">
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>60</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>70</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>90</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="runMeanshiftButton">
      <property name="text">
       <string>Start Meanshift !</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
